extension/src/popup/components/sendPayment/SendConfirm/TransactionDetails/index.tsx

```diff
@@ -120,6 +125,60 @@ const TwoAssetCard = ({
   );
 };
 
+const computeDestMinWithSlippage = (
+  slippage: string,
+  destMin: string,
+): BigNumber => {
+  const mult = 1 - parseFloat(slippage) / 100;
+  return new BigNumber(destMin).times(new BigNumber(mult));
+};
+
+const getOperation = (
```

Comment:
I pulled these out of the component body and moved all closure variables to fn args in order to avoid the re-renders that were caused by the closure style before, since this is now needed in an effect to compute the blocked account checks.